packages/config/src/load.js

```diff
@@ -8,6 +8,14 @@ import destr from 'destr'
 import * as rc from 'rc9'
 import { defaultNuxtConfigFile } from './config'
 
+function coalesce (...values) {
```

Comment:
Might be worth adding a comment above explaining what this function does ☺️

Reply:
```suggestion
// Returns the first value that isn't null or undefined (as replacement for ??)
function coalesce (...values) {
```